support/modules/single-file

```diff
@@ -28,7 +29,7 @@ case "$STATE" in
         mkdir -p $tmp_dest_dir
         if test -f ${dest_dir}/${filename}
         then
-            if ! cp ${dest_dir}/${filename} ${tmp_dest_dir}
+            if ! cp -a ${dest_dir}/${filename} ${tmp_dest_dir}
```

Comment:
Why do we need these?

Reply:
Normal `cp` is not guaranteed to preserve all file attributes (umask can interfere). This makes sure that the backup has exactly the same file attributes as the original.